src/bio/bio_internal.h

```diff
@@ -31,6 +30,14 @@
 #define NVME_MONITOR_PERIOD	    (60ULL * (NSEC_PER_SEC / NSEC_PER_USEC))
 #define NVME_MONITOR_SHORT_PERIOD   (3ULL * (NSEC_PER_SEC / NSEC_PER_USEC))
 
+static bool bio_vmd_enabled;
```

Comment:
Is it ok to define a variable in header file?

Reply:
fixed